codegenerator/cli/templates/static/codegen/src/eventFetching/ChainFetcher.res

```diff
@@ -163,18 +163,18 @@ let makeFromConfig = (chainConfig: Config.chainConfig, ~maxAddrInPartition) => {
 /**
  * This function allows a chain fetcher to be created from metadata, in particular this is useful for restarting an indexer and making sure it fetches blocks from the same place.
  */
-let makeFromDbState = async (chainConfig: Config.chainConfig, ~maxAddrInPartition) => {
+let makeFromDbState = async (chainConfig: Config.chainConfig, ~maxAddrInPartition, ~sql=Db.sql) => {
```

Comment:
Is there any significance in passing the sql as a parameter here?

Reply:
This is to pass a transaction context in tests, so I can rollback dynamic contracts after they are deleted

Reply:
Ok, it is to make it easier to test I assume - nice :+1: